packages/react-components/react-positioning/src/usePositioning.ts

```diff
@@ -267,17 +267,21 @@ function useContainerRef(updatePosition: () => void, enabled: boolean) {
       Object.assign(container.style, { position: 'fixed', left: 0, top: 0, margin: 0 });
     }
 
-    toggleScrollListener(container, prevContainer, updatePosition);
-
-    updatePosition();
+    // toggleScrollListener requires computed styles; thus use RAF to prevent forced style reevaluation.
+    window.requestAnimationFrame(() => {
+      toggleScrollListener(container, prevContainer, updatePosition);
+      updatePosition();
+    });
   });
 }
 
 function useTargetRef(updatePosition: () => void) {
   return useCallbackRef<HTMLElement | PositioningVirtualElement | null>(null, (target, prevTarget) => {
-    toggleScrollListener(target, prevTarget, updatePosition);
-
-    updatePosition();
+    // toggleScrollListener requires computed styles; thus use RAF to prevent forced style reevaluation.
+    window.requestAnimationFrame(() => {
```

Comment:
All `requestAnimationFrame()` calls should be cleaned up on a component to avoid reference/memory leaks or throwing exceptions. Typically it's:

```ts
const requestRef = React.useRef();

// ---
requestRef.current = requestAnimationFrame();
// ---

React.useEffect(() => {
  return () => {
    cancelAnimationFrame(requestRef.current);
  };
}, []);
```